scripts/typescript/package.json

```diff
@@ -1,21 +1,26 @@
 {
   "name": "cript-typescript",
   "version": "1.0.0",
-  "description": "Set of typescript code to convert raw data (*.csv, *.ts) to CRIPT *.json format",
+  "description": "Set of typescript code to convert raw data (*.xlsx, *.csv, *.ts) to CRIPT *.json format",
   "main": "index.js",
   "scripts": {
-    "afrl": "npx ts-node ./src/afrl/index.ts",
-    "rcbc": "npx ts-node ./src/rcbc/index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "npx tsc",
+    "afrl": "npx ts-node ./src/afrl",
+    "bcdb": "npx ts-node ./src/bcdb",
```

Comment:
I was working on two datasets at the time, that's why you can see some traces of it. Another PR is coming.